platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationFilesPoller.java

```diff
@@ -94,6 +97,7 @@ public void init() throws IOException {
     public void register(@NotNull ChangeListener listener) {
         notNull(listener, "ChangeListener cannot be null");
         changeListener = listener;
+        processExistingConfigurationFiles();
```

Comment:
The plan is to replace this with a better file poller that takes care of any existing files and any new files that get added afterward without having to worry about potential timing or synchronization problems.

Having said that, would this fix be acceptable for now?